frontend-wasm/src/miden_abi/mod.rs

```diff
@@ -0,0 +1,40 @@
+pub(crate) mod transform;
+pub(crate) mod tx_kernel;
+
+use miden_core::crypto::hash::RpoDigest;
+use miden_hir::{FunctionType, Symbol};
+
+/// Parse the stable import function name and the hex encoded digest from the function name
+pub fn parse_import_function_digest(import_name: &str) -> Result<(String, RpoDigest), String> {
+    // parse the hex encoded digest from the function name in the angle brackets
+    // and the function name (before the angle brackets) example:
+    // "miden:tx_kernel/note.get_inputs"
+    let mut parts = import_name.split('<');
```

Comment:
Perhaps we need to use `rsplit` here so that we don't run into issues with function names that have brackets in them (i.e. due to generics)? Maybe that's not a practical issue here, but just something that came to mind while glancing through this function.

Reply:
`import_name` is the string that we put in `link_name` annotation at
https://github.com/0xPolygonMiden/compiler/blob/cfabce971c3357f2feb3d559406be3ea97890468/sdk/tx-kernel/src/lib.rs#L20-L26 and not a real function name. I believe we're safe as long as we control `link_name` content.